python/src/rest_api/aac_rest_app.py

```diff
@@ -44,7 +38,12 @@ def _get_files_in_context() -> list[AaCFile]:
     Returns:
         A list of all files contributing definitions to the active context.
     """
-    return list({definition.source for definition in ACTIVE_CONTEXT.definitions})
+    global ACTIVE_CONTEXT
+    if not ACTIVE_CONTEXT:
+        ACTIVE_CONTEXT = LanguageContext()
```

Comment:
I don't think this is the right move. I think it would just be created each time in a normal implementation stance.
```suggestion
    active_context = LanguageContext()
```